components/Snackbar/tests/unit/MDCSnackbarMessageViewTests.m

```diff
@@ -123,7 +123,42 @@ - (void)testAccessibilityHintSetFromSnackbarMessageProperty {
                         self.message.accessibilityHint);
 }
 
-- (void)testSnackbarSetAccessibiltyViewIsModalForActionSnacbars {
+- (void)testSnackbarAccessibiltyViewIsModalByDefaultToNoWithActions {
```

Comment:
I think we spoke about the naming here. I believe we settled on something like `testSnackbarAccessibiltyViewIsModalDefaultsToNoWithActions`?

Reply:
Addressed in a5ddbbd01590490c2cebea9e302ad8fbdd774c28